### ISSUE TITLE
Cashew crashes all the time

### ISSUE THREAD

**Issue by ankushdharkar** [simplerocket-llc/dear-cashew#84]
<img width="470" alt="screen shot 2017-04-18 at 3 15 30 pm" src="https://cloud.githubusercontent.com/assets/1935403/25155467/f3386d60-2449-11e7-8dc9-6445db6a9c7a.png">

I have done the following: 
- Uninstalled Cashew using app cleaner
- Installed from the app store
- Signed-in and waited for the repositories to sync

**Comment by victoriafrench** [comment]
Same issue here. I was a first time install, signed in and was waiting for sync to complete (hours of spinny wheel of death). App just crashes on launch now. Window appears, vanishes in a blink and crash reporter opens.

**Comment by jhoek** [comment]
Yep, more or less the same here - endless beachball of death. Sad, because the app looks really promising!
